src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Random random = new Random();
+
+    public static Colors getRandomColor() {
```

Comment:
Method and field should not be static
```suggestion
    private final Random random = new Random();

    public Colors getRandomColor() {
```